requirements.txt

```diff
@@ -1,7 +1,7 @@
 # Neuro SAN
-neuro-san==0.5.35
+neuro-san==0.5.36
 neuro-san-web-client==0.1.12
-nsflow==0.5.12
+nsflow==0.5.13
```

Comment:
Does it make sense to get the version of neuro-san from nsflow and omit it here?

Reply:
`nsflow` does not require `neuro-san` for now. So no.